src/Process/PointCloudProcessing.js

```diff
@@ -163,18 +163,49 @@ function markForDeletion(elt) {
 }
 
 export default {
-    preUpdate(context, layer) {
-        // TODO: use changeSource
-        layer.counters = {
-            pointCount: 0,
```

Comment:
I see that the `pointcloud.js` and `pointcloud_globe.js` examples refer to `pointcloud.counters.pointCount`. Wouldn't that be a problem if you removed that count?

Reply:
Also, if you just keep `displayedCount` you may want to go with `layer.displayCount` instead of having an intermediary `layer.counters` object.

Reply:
Fixed.